ephios/extra/templates/extra/widgets/recurrence_picker.html

```diff
@@ -0,0 +1,219 @@
+{% load i18n %}
+{% load static %}
+
+<input type="hidden" autocomplete="off" id="pick_hour" value="{{ widget.pick_hour|lower }}">
+<input type="hidden" autocomplete="off" id="original_start" value="{{ widget.original_start }}">
+<div id="vue_app" class="row mt-4">
+    <div id="chooser" class="col-12 col-md-6 mb-2">
+        <div class="input-group pb-4">
+            <div class="input-group-text">{% translate "starting at" %}</div>
+            <input type="date" class="form-control" v-model="DTSTART">
+            <input type="time" class="form-control" v-if="pickHour" v-model="DTSTART_TIME">
+        </div>
+        <ul class="list-group mb-4">
+            <li class="list-group-item p-3" v-for="(rule, index) in rules"
+                :class="{'border-danger': !isRuleValid(rule)}">
+                <div class="btn-toolbar justify-content-between overflow-auto">
+                    <div class="btn-group pb-4">
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===4}" v-if="pickHour"
+                                @click="changeFreq(rule, frequencies.HOURLY)">{% translate "Hourly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===3}"
+                                @click="changeFreq(rule, frequencies.DAILY)">{% translate "Daily" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===2}"
+                                @click="changeFreq(rule, frequencies.WEEKLY)">{% translate "Weekly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===1}"
+                                @click="changeFreq(rule, frequencies.MONTHLY)">{% translate "Monthly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===0}"
+                                @click="changeFreq(rule, frequencies.YEARLY)">{% translate "Yearly" %}</button>
```

Comment:
i mean I can use inputs instead of buttons if you prefer that, but I need to reset stuff based on the new frequency anyways, so won't change anything in regards to the function calls